src/main/java/ladder/domain/Floor.java

```diff
@@ -0,0 +1,57 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.List;
+import java.util.Random;
+
+public class Floor {
+    private final static int INIT_SECTOR_NUMBER = 0;
```

Comment:
Class and member modifiers, when present, appear in the order recommended by the Java Language Specification:

```
public protected private abstract default static final transient volatile synchronized native strictfp
```